automation/update-from-application-services.py

```diff
@@ -0,0 +1,107 @@
+#!/usr/bin/python3
+
```

Comment:
nit: since this essentially replaces `update_package_swift.py` we probably want to replace it here as well?
https://github.com/mozilla/rust-components-swift/blob/e8753f796db870ef36664bf476e82f97710e9bc2/.github/workflows/update-as.yml#L50-L55

Or remove that block entirely

Reply:
I think this entire workflow isn't going to work with the new system, so I deleted it for now.  I'd like to run the process manually a couple times, then we can add something like this back.